dpnp/backend/tests/test_broadcast_iterator.cpp

```diff
@@ -158,8 +105,12 @@ TEST_P(IteratorBroadcasting, sycl_broadcast)
 
     for (dpnpc_index_t i = 0; i < result_size; ++i)
     {
-        EXPECT_EQ(result.at(i), param.result.at(i));
+        EXPECT_EQ(result[i], param.result.at(i));
     }
+
+    input_it->~DPNPC_id();
```

Comment:
no need to explicitly call the destructor.

Reply:
Needed for `placement new`.